packages/browser/package.json

```diff
@@ -5,18 +5,27 @@
   "types": "dist/index.d.ts",
   "browser": "dist/index.js",
   "main": "dist/index.js",
+  "module": "dist/index.mjs",
   "bundle": "dist/solid-client-authn.bundle.js",
   "repository": {
     "url": "https://github.com/inrupt/solid-client-authn"
   },
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    }
+  },
   "scripts": {
     "prepublishOnly": "npm run build",
     "clean": "npm run clean-module && npm run clean-browser && rimraf ./coverage",
     "clean-module": "rimraf ./dist",
     "clean-browser": "rimraf ./browserDist",
-    "build": "npm run build-module && npm run build-browser",
+    "build": "npm run build-module && npm run build-browser && npm run build-esm",
```

Comment:
Let me look into more details about what `build-module` does, but it sounds that it should build ESM too.

Reply:
I think it is just badly named since it is just executing `tsc` with [this config](https://github.com/inrupt/solid-client-authn-js/blob/main/packages/browser/tsconfig.json).

Reply:
Then I assume it would make sense to also collapse all bundling into rollup, and output both CJS and ESM?